pre_commit_hooks/pretty_format_json.py

```diff
@@ -69,8 +69,8 @@ def pretty_format_json(argv=None):
     parser.add_argument(
         '--indent',
         type=parse_indent,
-        default='  ',
-        help='String used as delimiter for one indentation level',
+        default=2,
+        help='String used as delimiter for one indentation level (Default: 2)',
```

Comment:
this shouldn't be refactored -- it allows for the following inputs:

- `'\t'`
- '  '
- '4'

See https://github.com/pre-commit/pre-commit-hooks/pull/120 which is essentially the opposite of this patch :)

Reply:
Hmm ok, strings break Py2 json. Suggestions? Keep simplejson and have json as fallback?

There is no need for that function tbh since both simplejson and py3 json accept both str and int and do the conversion...

Reply:
The original code never allowed `'\t'`. `'\t'.strip() == '\t'`

`''` and `'4'` are still value and coverted to 0 and 4 ints.

Reply:
Yes and `\t` raised a valueerror with original code.

Reply:
Ok what I suggest is to leave the default as an int and just pass the values `str` or `int` to json and catch any exceptions.

Reply:
```python
>>> '\t'.strip()
''
```

hmmm seems to do what I expect :)

Reply:
Oh I see you edited, disregard :D

Reply:
Yeah I was testing with strip(' ') not strip() :scream: 

I think I have the solution for all this and will update...